.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+	"useTabs": true,
+	"singleQuote": false,
+	"trailingComma": "none",
+	"printWidth": 100,
+	"plugins": ["prettier-plugin-svelte"],
+	"pluginSearchDirs": ["."],
+	"overrides": [{ "files": "*.svelte", "options": { "parser": "svelte" } }]
+}
```

Comment:
I would highly prefer to keep default Prettier config.

Reply:
default Prettier config means removing .prettierrc file. Please remove it completely and run prettier again. thank you